CONTRIBUTING.md

```diff
@@ -85,6 +86,17 @@ If you believe you've found a security vulnerability in one of our products or p
 - A detailed description of the steps required to reproduce the vulnerability (proof of concept source code, screenshots, and compressed screen captures are all helpful to us); and
 - Your name/handle and a link for recognition in our Hall of Fame.
 
+### Improving our documentation
+
+**Note:** medic-docs does not involve release management and acceptance testing. Help us maintain the quality of our documentation by submitting a pull request (PR) with any suggested changes. 
+
+Is our documentation up to date? Have we covered everything we should? Could our wording be improved? Read our [Documentation Style Guide](https://github.com/medic/medic-docs/blob/master/development/docs-style-guide.md) then open a pull request with your suggested changes or additions.
+Want to talk about Documentation generally? Join our [Community Forum](https://communityhealthtoolkit.org/forum)!
+
+### Code of Conduct
+
+All maintainers and contributors are required to act according to our [Code of Conduct](https://github.com/medic/medic/blob/master/CODE_OF_CONDUCT.md). Thank you for your help building a positive community and a safe environment for everyone.
+=======
```

Comment:
It looks like this line was left in by mistake.

Reply:
So sorry, which line? My eyes are crossing from editing decks today!

Reply:
@CescaGarrett Sorry for the late response - I missed your comment somehow. The `=====` line looks like it was left in from the merge conflict resolution.

Reply:
@CescaGarrett Did you see my comment? I'd love to get this merged - let me know if I can help.

Reply:
Done!